<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1177</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>701</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_dataset_path">
       <property name="text">
        <string>Dataset Path：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_dataset_path">
       <property name="text">
        <string>F:/Dataset</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>441</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_dataset">
       <property name="text">
        <string>Dataset：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_dataset"/>
     </item>
     <item>
      <widget class="QLabel" name="label_type">
       <property name="text">
        <string>QA Type：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_type"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_question">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>170</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>Question：</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_answer">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>280</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>Answer：</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>330</y>
      <width>491</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_A">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_B">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_C">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_D">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_E">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_refresh">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>92</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_search">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>92</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>640</width>
       <height>360</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>vqa.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_callback">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>92</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Callback</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_question">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>210</y>
      <width>491</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>150</y>
      <width>20</width>
      <height>401</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>140</y>
      <width>1151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>label_dataset_path</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
   <zorder>label_type</zorder>
   <zorder>label_type</zorder>
   <zorder>label_question</zorder>
   <zorder>label_answer</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>pushButton_refresh</zorder>
   <zorder>pushButton_search</zorder>
   <zorder>widget</zorder>
   <zorder>pushButton_callback</zorder>
   <zorder>textBrowser_question</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1177</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
